README.md

```diff
@@ -17,19 +17,19 @@ The default infrastructure consists of the following components:
 
 |Host|IP address|Role|
 |-|-|-|
-|omserver|192.168.1.100|Ops Manager server + Application DB|
-|n\[1-2\]|192.168.1.10\[1-2\]|client with Automation installed|
-|bkp|192.168.1.103|S3 storage|  
-|provisioner*|192.168.1.99|Ansible provisioner|
-|ldapserver*|192.168.1.104|OpenLDAP server|
-|kmip*|192.168.1.105|KMIP server|
+|omserver.om_ansible_front|192.168.1.100|Ops Manager server + Application DB|
+|n\[1-2\].om_ansible_front|192.168.1.10\[1-2\]|client with Automation installed|
```

Comment:
We are missing the existence of `n3` so the change will be: `n\[1-3\].om_ansible_front`